blockly-core/core/ui/contract_editor/svg_highlight_box.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+goog.provide('Blockly.SvgHighlightBox');
+
+/**
+ * A horizontal SVG highlight box with rectangular background and text
+ * @param {!Element} parent {!Element}
+ * @param {Object=} opt_options
+ * @constructor
+ */
+Blockly.SvgHighlightBox = function (parent, opt_options) {
+  var color = opt_options.color || '#000';
+  var thickness = opt_options.thickness || '30';
+
+  var extraStyle = 'pointer-events: none;';
+
+  this.svgGroup_ = Blockly.createSvgElement('g', {style: extraStyle},
+    parent);
+  this.highlightRectangle_ = Blockly.createSvgElement('rect',
+    {'fill': 'none', 'stroke-width': thickness, 'stroke': color}
+    , this.svgGroup_);
```

Comment:
not sure if our style docs say this explicitly, but i prefer to split out objects like this with one item per line. I would also have the comma at the end of the preceeding line